src/Compilers/CSharp/Portable/Binder/Binder_Constraints.cs

```diff
@@ -67,7 +67,7 @@ internal partial class Binder
                     Debug.Assert(ordinal < n);
 
                     (TypeParameterConstraintClause constraintClause, ArrayBuilder<TypeConstraintSyntax>? typeConstraintNodes) =
-                        this.BindTypeParameterConstraints(typeParameterList.Parameters[ordinal], clause, isForOverride, canIgnoreNullableContext: canIgnoreNullableContext, diagnostics);
+                        this.BindTypeParameterConstraints(typeParameterList.Parameters[ordinal], clause, isForOverride, canUseLightweightTypeConstraintBinding, diagnostics);
```

Comment:
>canUseLightweightTypeConstraintBinding [](start = 120, length = 38)

Consider keeping the named argument since there are two `bool` parameters in a row.